tests/testem_connection_tests.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const patchEmitterForWildcard = require('../public/testem/testem_connection');
+const expect = require('chai').expect;
+const ServerIO = require('socket.io');
+const ClientIO = require('socket.io-client');
+
+describe.only('Testem Connection', function() {
+  var serverIo, client;
+
+  function replaceGlobals(newGlobals, originalGlobals) {
```

Comment:
I feel we should restore the globals in the after() hook, prevent any gotchas.

Reply:
Agree.